### ISSUE TITLE
ERROR p2p: could not retrieve subnets: missing ENR key "attnets"

### ISSUE THREAD

**Issue by q9f** [prysmaticlabs/prysm#5924]
# 🐞 Bug Report

Hello, the following p2p error makes the logs unreadable. 

### Description

```
[2020-05-20 12:08:49] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:08:49] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:08:49] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:08:49] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:08:49] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:08:49] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:08:49] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:08:49] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
[2020-05-20 12:09:00] ERROR p2p: could not retrieve subnets: missing ENR key "attnets"
```

### Has this worked before in a previous version?

Yes.

## 🔬 Minimal Reproduction

Seeing this on all my Schlesi and Witti nodes.

## 🔥 Error

![Screenshot at 2020-05-20 12-10-13](https://user-images.githubusercontent.com/58883403/82434295-f2585780-9a92-11ea-9c24-7ebfd5871260.png)

It would be nice to either fix this or move it to debug level. :pray:

**Comment by nisdas** [comment]
@q9f Can you pull in the latest ? The fix was just merged in today #5917. I am closing this now, if you still see it please reopen the issue.

**Comment by q9f** [comment]
Right, thanks!
